services/sidecar/docker/boot.sh

```diff
@@ -32,15 +32,28 @@ then
   DEBUG_LEVEL=info
 fi
 
-
 # RUNNING application ----------------------------------------
-if [[ ${SC_BOOT_MODE} == "debug" ]]
+
+# default
+DEBUG_LEVEL=info
+CONCURRENCY=2
+POOL=prefork
+
+
+if [[ ${SC_BOOT_MODE} == "debug-ptvsd" ]]
+then
+  # NOTE: in this case, remote debugging is only available in development mode!
+  # FIXME: workaround since PTVSD does not support prefork subprocess debugging: https://github.com/microsoft/ptvsd/issues/943
+  POOL=solo
+
+elif [[ ${SC_BOOT_MODE} == "debug" ]]
 then
-  # TODO: activate pdb??
   DEBUG_LEVEL=debug
   CONCURRENCY=1
```

Comment:
would be nice to have this as a environment variable. But we are going to refactor the whole thing anyhow. Sometime. If we find time. Maybe.

Reply:
This is indeed the case with the new template produced with the cookie-cutter